docs/user_guide/admin_guide/deployment/dashboard_ui.rst

```diff
@@ -20,7 +20,7 @@ for client sites and then download startup kits for each client site after it is
 Finally, the ``Project Admin`` or project administrator is responsible for overall site management from setting up the project
 to approving the users and client sites after inviting users to sign up (for details on how the project administrator can set up
 the site, see :ref:`dashboard_api`). The ``Project Admin`` can also download startup kits
-for the server(s) and overseer (if :ref:`HA mode <high_availability>` is enabled).
+for the server(s).
```

Comment:
now we don't have overseer, I think we don't support multiple servers now, @IsaacYangSLA can you confirm? thanks